daily-challenges/2024-09/28/thaoha93.spec.ts

```
@@ -0,0 +1,30 @@
+function isPrime(num: number) {
+    if (num <= 1) {
+        return false;
+    }
+    for (let i = 2; i <= num / 2; i++) {
+        if (num % i == 0) {
+            return false;
+            break;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)